src/components/AppVersionRedirectDialog.svelte

```diff
@@ -0,0 +1,52 @@
+<!--
+  (c) 2023, Center for Computational Thinking and Design at Aarhus University and contributors
+ 
+  SPDX-License-Identifier: MIT
+ -->
+
+<script lang="ts">
+  import { t } from '../i18n';
+  import { hasSeenAppVersionRedirectDialog } from '../script/stores/uiStore';
+  import StandardDialog from './dialogs/StandardDialog.svelte';
+  import StandardButton from './StandardButton.svelte';
+
+  let isOpen = true;
+
+  const appVersionRedirect = () => {
+    hasSeenAppVersionRedirectDialog.set(true);
+    window.location.href = 'https://ml.microbit.org';
```

Comment:
Let's not rely on the redirect at the root, which could change in future.

```suggestion
    window.location.href = 'https://ml.microbit.org/thenextgen/';
```